src/browser.qml

```diff
@@ -21,6 +21,7 @@ ApplicationWindow {
     id: window
 
     readonly property bool largeScreen: Screen.sizeCategory > Screen.Medium
+    readonly property int browserVisibility: _mainWindow && _mainWindow.visibility || QuickWindow.Window.Hidden
```

Comment:
_mainWindow.visibility || QuickWindow.Window.Hidden looks weird?

Reply:
"_mainWindow ? _mainWindow.visibility : QuickWindow.Window.Hidden" perhaps?

Reply:
I tend to use that notation. Will check to ternary operator.